cnf-tests/README.md

```diff
@@ -369,7 +369,7 @@ Various tests have different requirements. Some of them:
 
 ### Limiting the nodes used during tests.
 
-The nodes on which the tests will be executed can limited by means of specifying a `NODES_SELECTOR` environment variable. Any resources created by the test will then be limited to the specified nodes.
+The nodes on which the tests will be executed can be limited by means of specifying a `NODES_SELECTOR` environment variable. Any resources created by the test will then be limited to the specified nodes.
```

Comment:
For clarity, maybe this could be reworked a bit to remove some of the passive voice:

```suggestion
Specifying a `NODES_SELECTOR` environment variable can limit the nodes which will be chosen for test execution. Any resources created by the test will then be limited to the specified nodes.
```

Reply:
/Suggestions
```suggestion
The nodes on which the tests will be executed can be limited by means of specifying a `NODES_SELECTOR` environment variable. Any resources created by the test will then be limited to the specified nodes.
```